wordpress-mobile/WordPress-iOS
# Reader: Enable Follow Conversation for DotCom and Jetpack sites

Ref. pbArwn-1n5-p2
Closes #15273 

This PR enables the "Follow Conversation" feature for all DotCom and Jetpack sites. This feature does not exist for external sites.

### To test
**Set up**
On the web or in the mobile app, ensure you have followed:
- a DotCom or Jetpack site
- an external site (self-hosted, not connected to Jetpack)

So that the "Following" tab inside of Reader is populated with posts.

**Test Case 1**
Follow Conversation on a DotCom or Jetpack site.

1. Navigate to Reader > Following tab.
2. Find a post where the Comments button is enabled.
3. Tap the Comments button.
4. In the Reader Comments view, ensure the Follow Conversation button is displayed.
5. Optional: subscribe / unsubscribe to see the button states and toast messages.

**Test Case 2**
1. Navigate to Reader > Following tab.
2. Find a post from an external site.
3. Notice the Comment button is disabled.

### Screenshots
| iPhone 11 Pro | iOS 14.2 |
| --- | --- |
| **Reader > Following** | **Comments view** |
|  ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-02 at 09 40 09](https://user-images.githubusercontent.com/1062444/100897650-8749fd80-3485-11eb-953d-add049c7e454.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-02 at 08 51 44](https://user-images.githubusercontent.com/1062444/100897669-8dd87500-3485-11eb-8e49-e7d0e50b71ad.png) |
| **Following** | **Unfollowing** | 
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-02 at 09 58 30](https://user-images.githubusercontent.com/1062444/100897786-af396100-3485-11eb-89e0-98c475e66960.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-02 at 09 58 40](https://user-images.githubusercontent.com/1062444/100897847-c11b0400-3485-11eb-80ae-e6a6b60365a5.png) |


PR submission checklist:
